be/src/runtime/exec_env_init.cpp

```diff
@@ -183,7 +183,6 @@ Status ExecEnv::_init_mem_tracker() {
                      << ". Using physical memory instead";
         global_memory_limit_bytes = MemInfo::physical_mem();
     }
-    MemTracker::get_process_tracker()->set_limit(global_memory_limit_bytes);
```

Comment:
so we do NOT limit total memory by process tracker, and it's just used to track virtual memory of process, right?

Reply:
yes, I modified the process tracker comment.